ui/app/models/auth-method.js

```diff
@@ -68,19 +68,29 @@ export default class AuthMethodModel extends Model {
     return this.local ? 'local' : 'replicated';
   }
 
+  userLockoutConfig = {
```

Comment:
Originally I defined these in the component above, but old ember components are strict about what can be defined as a default and it didn't like objects or arrays as default variable values. Since I wanted to access these in the template, I defined them here.